File: ios/BFExplorer/Plaoc/IPCCore/IPC/StreamAsRawData.swift
//
//  StreamAsRawData.swift
//  BFExplorer
//
//  Created by ui03 on 2023/3/15.
//

import UIKit

class StreamAsRawData {
    
    var stream_id: String
    var stream: InputStream
    var ipc: Ipc
    
    init(stream_id: String, stream: InputStream, ipc: Ipc) {
        self.stream_id = stream_id
        self.stream = stream
        self.ipc = ipc
    }
}

File: android/app/src/main/java/info/bagen/dwebbrowser/microService/core/ipc/StreamAsRawData.kt
package info.bagen.dwebbrowser.microService.core.ipc

import java.io.InputStream

/**
 * 如果 rawData 是流模式，需要提供数据发送服务
 *
 * 这里不会一直无脑发，而是对方有需要的时候才发
 * @param stream_id
 * @param stream
 * @param ipc
 */
class StreamAsRawData(
    val stream_id: String,
    val stream: InputStream,
    ipc: Ipc
) {
//    stream
}